projects/components/src/overlay/sheet/sheet-overlay.component.ts

```diff
@@ -46,6 +60,8 @@ export class SheetOverlayComponent {
   public readonly rendererInjector: Injector;
   public visible: boolean = true;
   public readonly closeOnEscape: boolean;
+  public readonly attachedTriggerTemplate?: TemplateRef<unknown>;
+  public isViewCollapsed: boolean;
```

Comment:
can we repurpose the `visible` property for this? both seems to be doing similar things.

Reply:
Visible is actually hiding the entire content (including the trigger) before the popover is closed. So, it is different than `isViewCollapsed` and I can't combine them. Also, keeping them separate keeps the blast radius small